hazelcast/src/test/java/com/hazelcast/test/compatibility/SamplingNodeExtension.java

```diff
@@ -294,16 +294,13 @@ public CPPersistenceService getCPPersistenceService() {
 
     @Override
     public JetService getJet() {
-        throw new IllegalStateException("Since we don't yet provide a compatibility guarantee between"
-                + " minor versions in the Jet classes, we don't run Jet tests to capture compatibility"
-                + " samples. Please exclude this Jet test from the sampling process.");
+        // There are 3 JET classes that must be backward compatible, so let's start the jet test too.
```

Comment:
The comment shouldn't be an answer to what was here before this PR. I'll move this comment below.
```suggestion
```